### ISSUE TITLE
Раздел "Расписание". При поиске кинотеатра по станции метро в список доступных кинотеатров добавляется кинотеатр, не отвечающий заданным требованиям к его местоположению (не располагается вблизи к выбираемой станции метро)

### ISSUE THREAD

**Issue by ArtemyBabitsky** [ArtemyBabitsky/Test-afisha.mail.ru-#4]
_Как воспроизвести:_
1. Открыть главную страницу сайта: https://afisha.mail.ru/spb/cinema/kinoafisha;
2. В поле "Метро" ввести название станции метро «Невский проспект»;
3. Отсортировать доступные фильмы по количеству сеансов;
4. Нажать кнопку «Расписание» в отношении фильма «Дом для странных детей мисс Перегрин»;
5. Просмотреть список доступных кинотеатров.

_Фактический результат:_
При сортировке по станции метро «Невский проспект» в расписании фильма выводится 2 доступных кинотеатра (1) «Cinema Grand Palace» – СПб,Итальянская д.15; 2) «Каскад» - СПб, г. Петергоф, Царицынская ул., д. 2). Кинотеатр №2 не находится вблизи станции метро «Невский проспект».
![4](https://cloud.githubusercontent.com/assets/23294267/20038967/6582474c-a44d-11e6-92e1-4278b4169fba.jpg)

_Ожидаемый результат:_
При сортировке по станции метро «Невский проспект» в расписании фильма выводятся доступные кинотеатры, расположенные в непосредственной близости к выбранной станции метро.

**Comment by marinapenkova** [comment]
Слишком уж длинное название. Есть такие баг-трекеры, в которых стоит ограничение на количество символов в названии ошибки

**Comment by ArtemyBabitsky** [comment]
@marinapenkova , я согласен, что название длинное, однако в более сжатом варианте однозначно передать "проблемность" бага у меня не получилось. Был бы Вам признателен, если бы предложили свой оптимальный вариант.

**Comment by marinapenkova** [comment]
Я бы написала так: "Лишний кинотеатр выводится на экран при поиске по
станции метро"
Остальное будет понятно из шагов и прикрепленной картинки
